people/sivadeilra.toml

```diff
@@ -0,0 +1,3 @@
+name = 'sivadeilra'
```

Comment:
This should be "Arlie Davis".

Reply:
Fixed it manually. `cargo run add-person` gets the name from Github.